client/state/lib/middleware.js

```diff
@@ -161,6 +162,12 @@ const handler = ( dispatch, action, getState ) => {
 			}, 0 );
 			return;
 
+		case INVITE_ACCEPTED:
```

Comment:
Could this be part of the `inviteAccepted` action creator instead? I'd very much like to avoid introducing new code to the globally available middleware when possible. Ideally, everything should be modular and no such middleware will be necessary.

Reply:
Let's do this when we decide to remove this middleware for good. Right now, I like that both `user().incrementSiteCount()` and `user().decrementSiteCount()` are nicely concentrated next to each other in one file.

The `lib/user` module is always present everywhere, so this middleware doesn't even introduce any unwanted dependencies.

Reply:
Sounds like a good plan 👍